boxsdk/client/client.py

```diff
@@ -585,6 +585,36 @@ def create_group(
             response_object=response,
         )
 
+    def task(self, task_id):
+        """
+        Initialize a :class:`Task` object, whose box id is task_id.
+
+        :param task_id:
+            The box ID of the :class:`Task` object.
+        :type task_id:
+            `unicode`
+        :return:
+            A :class:`Task` object with the given entry ID.
+        :rtype:
+            :class:`Task`
+        """
+        return self.translator.translate('task')(session=self._session, object_id=task_id)
```

Comment:
It would be nice to switch these sorts of usages over to use `translator.get()` instead — it does the same thing, and will be forwards-compatible with the nested object translation work I'm doing